packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/permission/Project.java

```diff
@@ -0,0 +1,20 @@
+package software.uncharted.terarium.hmiserver.models.dataservice.permission;
+
+public class Project {
```

Comment:
To avoid confusion should this be named `ProjectPermission` or something similar? I'm not sure if having `software.uncharted.terarium.hmiserver.models.dataservice.permission.Project` and `software.uncharted.terarium.hmiserver.models.dataservice.Project` will be clear